packages/components/src/local/molecules/admin-message/types/props.d.ts

```diff
@@ -0,0 +1,16 @@
+import { Message } from '@serge/custom-types'
+
+export default interface PropTypes {
+  /**
+   * The message content
+   */
+  message: Message
+  /**
+   * Current force
+   */
```

Comment:
@zukilover  - I may be mistaken, but it looks like only `force.name` is used from the `force` component.  I suspect we could just use: `force: string`, and use the string directly in the admin-message element.

Reply:
I've just fixed this here: https://github.com/serge-web/serge/pull/497/commits/76dd3e1e3e60cbd783de74e83e2a55f79badec78

Reply:
looks good to me, you're a hero!